mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java

```diff
@@ -445,15 +445,30 @@ private HibernateOrmMapperOutboxPollingSettings() {
 			PREFIX + Radicals.COORDINATION_ENTITY_MAPPING_AGENT_TABLE;
 
 	/**
-	 * The name of the identifier generator used for the agent table.
+	 * The name of UUID generator strategy to be used by {@link UUIDGenerator}.
```

Comment:
```suggestion
	 * The name of UUID generator strategy to be used by {@link UUIDGenerator} for the agent table.
```

You might want to clarify for the corresponding outbox event configuration property as well?